include/zephyr/drivers/espi/espi_utils.h

```diff
@@ -5,7 +5,7 @@
  */
 
 /**
- * @file Header with definitions for eSPI drivers callback functions
+ * Header file with definitions for eSPI drivers callback functions
  */
 
 #ifndef ZEPHYR_DRIVERS_ESPI_UTILS_H_
```

Comment:
Why move this file? This doesn't seem to be needed as a publicly facing API

Reply:
Public API doesn't require it, but the emul subsys do (espi_emul_host). I moved it to be able to include using directive
`#include <drivers/espi/espi_utils.h>`
instead of
`#include "../../../drivers/espi/espi_utils.h"`
since the root of source code is not in the include paths